pkg/config/config.go

```diff
@@ -64,9 +64,10 @@ func Get() *Configuration {
 }
 
 func readConfigFile(v *viper.Viper) {
-	v.SetConfigName("config")
+	v.SetConfigName("config.yaml")
 	v.SetConfigType("yaml")
 	v.AddConfigPath("./configs/")
+	v.AddConfigPath("../../configs/")
```

Comment:
I'm not sure reading this config ever worked? Unless I'm breaking something here I don't know about. These changes let me use this locally.

Reply:
what didn't work without these changes?

Reply:
running tests using the variables in config file

Reply:
how did you run the test?

Reply:
`go test ./pkg/config/ `  seems to work fine for me without these two lines.

Reply:
hmm, I'm not sure why that behaves differently, but if you run `go test ./...`, tests that require db credentials will fail.

Reply:
oh i see!  yes, our make file is specifying CONFIG_PATH="/home/jlsherri/git/content-sources-backend/configs/"  which makes it work, but go test by itself didn't.  +1 to this change!

Reply:
Oh I think it's different because it sets an environment variable, so it's not reading from the config